docker/transformers-pytorch-gpu/Dockerfile.amd

```diff
@@ -3,27 +3,29 @@ LABEL maintainer="Hugging Face"
 
 ARG DEBIAN_FRONTEND=noninteractive
 
-RUN apt update
-RUN apt install -y git libsndfile1-dev tesseract-ocr espeak-ng python3 python3-pip ffmpeg
-RUN python3 -m pip install --no-cache-dir --upgrade pip
+RUN apt update && \
+    apt install -y --no-install-recommends git libsndfile1-dev tesseract-ocr espeak-ng python3 python3-pip ffmpeg && \
+    apt clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN python3 -m pip install --no-cache-dir --upgrade pip setuptools ninja git+https://github.com/facebookresearch/detectron2.git pytesseract "itsdangerous<2.1.0"
```

Comment:
wondering if there is any reason to move this to here?

Reply:
This avoids calling `RUN` multiple times for `pip install` to avoid having too many layers in the Dockerfile, maybe it is just taste though https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers